src/macaron/__main__.py

```diff
@@ -244,14 +233,14 @@ def main() -> None:
     )
 
     # Dump the default values.
-    sub_parser.add_parser(name="dump_defaults", description="Dumps the defaults.ini file to the output directory.")
+    sub_parser.add_parser(name="dump-defaults", description="Dumps the defaults.ini file to the output directory.")
 
-    # Verifying a provenance against a policy.
-    verify_parser = sub_parser.add_parser(name="verify")
+    # Verify the Datalog policy.
+    vp_parser = sub_parser.add_parser(name="verify-policy")
 
-    verify_parser.add_argument(
-        "-pr", "--provenance", required=True, type=str, help=("The path to the provenance file.")
-    )
+    vp_parser.add_argument("-d", "--database", required=True, type=str, help="Path to the database.")
+    vp_parser.add_argument("-f", "--file", required=False, type=str, default="", help="Path to the Datalog policy.")
```

Comment:
This flag is not required. However, in this method https://github.com/oracle-samples/macaron/blob/8aa0d6769c01a809a0f381950209750583ba92d1/src/macaron/policy_engine/policy_engine.py#L106, we are still treating it as not `None`. A file `open(...)` with the default value `""` would return a `FileNotFound` error. 

To replicate, we can run this command:
```
macaron verify-policy -d <path_to_dabase>
```

Reply:
See commit [c09c5a3](https://github.com/oracle-samples/macaron/pull/192/commits/c09c5a3379ef0733cd926c0b767480d77809a220).

Reply:
I wonder why don't we make the `file` argument `required=True` instead? We always exit when the user doesn't provide the path to the file so I think it's better if we make it mandatory.

Reply:
> I wonder why don't we make the `file` argument `required=True` instead? We always exit when the user doesn't provide the path to the file so I think it's better if we make it mandatory.

Good point.

Reply:
See commit [f58d698](https://github.com/oracle-samples/macaron/pull/192/commits/f58d6980b5b7c6649d07d753ced78ed31126cb19).